eva_submission/ena_retrieval.py

```diff
@@ -0,0 +1,120 @@
+import os.path
+
+import requests
+from ebi_eva_common_pyutils.config import cfg
+from ebi_eva_common_pyutils.logger import logging_config as log_cfg
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query, execute_query
+
+from eva_submission.assembly_taxonomy_insertion import download_xml_from_ena
+
+logger = log_cfg.get_logger(__name__)
+
+
+def files_from_ena(search_term):
+    xml_root = download_xml_from_ena(f'https://www.ebi.ac.uk/ena/browser/api/xml/textsearch?result=analysis&query={search_term}')
```

Comment:
Since this involves network calls, suggest using the usual retry mechanism that we use in other places in the codebase..

Reply:
Done